cmd/restoreoperator/main.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	"github.com/coreos/etcd-operator/pkg/controller/restorecontroller"
 	"github.com/coreos/etcd-operator/pkg/util/k8sutil"
-	version "github.com/coreos/etcd-operator/version/etcd-restore-operator"
+	version "github.com/coreos/etcd-operator/version/restoreoperator"
```

Comment:
restore-operator is probably cleaner.

Reply:
So some dirs are "restoreoperator" but some are "restore-operator".
Isn't it better to unify them?
I have no preference on either.